README.md

```diff
@@ -36,6 +36,27 @@ Check [dbt Hub](https://hub.getdbt.com) for the latest installation
 instructions, or [read the docs](https://docs.getdbt.com/docs/package-management) 
 for more information on installing packages.
 
+## Updating dbt-utils submodule
+
+If new version of dbt-utils is released, and you want to use it, update instruction below.
+Remember to always point HEAD to certain tag in this submodule, never leave HEAD pointing to any branch.
+
+1. `cd dbt-utils` go to dbt-utils submodule directory
+2. `git switch main` switch to main branch
+3. `git pull` pull changes from dbt-utils remote
```

Comment:
nit: change to `git fetch --tags`

Reply:
I used `pull` to fetch commits and tags and update local main branch with that commits at the same time, but sure we could change to `git fetch --tags`. Then local main branch wouldn't be updated, but maybe it doesn't matter as we are pointing anyway to specific commit through tag and leaving HEAD detached.